pkgs/by-name/su/supermariowar/package.nix

```diff
@@ -0,0 +1,77 @@
+{ lib
+, stdenv
+, fetchFromGitHub
+, cmake
+, pkg-config
+, enet
+, yaml-cpp
+, SDL2
+, SDL2_image
+, SDL2_mixer
+, zlib
+}:
+
+stdenv.mkDerivation rec {
```

Comment:
It's recommended to use `finalAttrs` here.

Reply:
Done :+1:.